tests/arch/ecp5/bug1836.mem

```diff
@@ -0,0 +1,32 @@
+0x8000,0x8324,0x8647,0x896a,0x8c8b,0x8fab,0x92c7,0x95e1,

```

Comment:
this is a large chunk of data of unclear copyright and origin, please replace with a placeholder

Reply:
This is taken from a random hex generator, is that placeholder enough?

Reply:
Correction, not random, it's a hex sine generator.  https://www.daycounter.com/Calculators/Sine-Generator-Calculator2.phtml